tests/test_config.py

```diff
@@ -0,0 +1,32 @@
+import json
+import shutil
+import unittest
+from pathlib import Path
+
+import ou_dedetai.config as config
+
+
+class TestConfigReadWrite(unittest.TestCase):
+    def setUp(self):
+        self.cfg = Path('tests/data/subdir/test_config.json')
+
+    @unittest.skip('TODO')
+    def test_update_config_file(self):
```

Comment:
Suggestion:

Perform operation with the code you want to test

Verify with a simple check (such as reading json)

Reply:
same comment as above: this file hasn't been updated yet for this PR